packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/data/TestIdentifiers.java

```diff
@@ -34,6 +34,7 @@ public final class TestIdentifiers {
     public static final UUID EPACKAGE_EMPTY_PORTAL_REPRESENTATION = UUID.fromString("05e44ccc-9363-443f-a816-25fc73e3e7f7");
 
     public static final UUID PORTAL_REPRESENTATION_DESCRIPTION = UUID.fromString("69030a1b-0b5f-3c1d-8399-8ca260e4a672");
+    public static final UUID PORTAL_REPRESENTATION_METADATA_ID = UUID.fromString("e81eec5c-42d6-491c-8bcc-9beb951356f8");
```

Comment:
This is the id of the portal representation which is already available in this file a couple line above. I'll update it